build.assets/macos/tsh/tsh.app/Contents/Info.plist

```diff
@@ -19,13 +19,13 @@
 		<key>CFBundlePackageType</key>
 		<string>APPL</string>
 		<key>CFBundleShortVersionString</key>
-		<string>17.0.6</string>
+		<string>1.0</string>
```

Comment:
This doesn't look right?

Reply:
I think this is a placeholder for all dev versions. @camscale is that right?

Reply:
Yes. The mac version number in this file does not support semver prerelease components - just 3 numbers. So all pre-releases get 1.0